modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/asset/entry/provider/test/AssetListAssetEntryProviderTest.java

```diff
@@ -1850,6 +1866,33 @@ private void _assertAssetListEntryResults(
 		}
 	}
 
+	private void _assertAssetListEntryResultsPagination(
+		AssetListEntry assetListEntry, long[] segmentsEntryIds,
+		AssetEntry... expectedAssetEntries) {
+
+		InfoPage<AssetEntry> infoPage =
+			_assetListAssetEntryProvider.getAssetEntriesInfoPage(
+				assetListEntry, segmentsEntryIds, null, null, StringPool.BLANK,
+				StringPool.BLANK, 0, 2);
```

Comment:
i think there are 3 results total? so maybe just double check that "0 to 2" doesnt actually return all 3 results..

Reply:
Part of this is ensuring that the second page returns the second page's results instead of the first page's again (as in the ticket).